packages/editor/src/components/widgets/code-editor/CodeEditor.css

```diff
@@ -25,5 +25,10 @@
   font-family: Consolas, 'Courier New', monospace;
   pointer-events: none;
   user-select: none;
-  z-index: 10;
+  z-index: 2;
+}
+
+.code-editor .with-lineNumbers {
+  left: 35px;
+  font-size: 12px;
```

Comment:
is the font-size different for full size monaco?

Reply:
Well not really. It is also 12px but for some reason the Placeholder-Text is way to big if I don't set the font-size.

with no font-size:
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/bc4aef5f-2699-4159-ae6e-9e810fcf59ef)
with font-size:
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/acee5ee3-850e-4dfe-81f0-7df620214c8c)

Reply:
Then I think you can set it to the default placeholder css class right? would make more sense to me.